autogen/main/outputs.tf.tmpl

```diff
@@ -147,6 +147,11 @@ output "dns_cache_enabled" {
   value       = local.cluster_dns_cache_enabled
 }
 
+output "cluster_gce_pd_csi_driver_enabled" {
+  description = "Whether GCE Persistent Disk CSI Driver enabled"
+  value       = local.cluster_gce_pd_csi_driver_enabled
```

Comment:
This should be a reference to the cluster itself, not merely passing back the input.

Or omit entirely, not sure this output is needed.

Reply:
You're right, i'm going to drop the output/local reference completely